src/components/datagrid/data_grid.test.tsx

```diff
@@ -2481,25 +2519,22 @@ describe('OuiDataGrid', () => {
       component.update();
 
       let focusableCell = getFocusableCell(component);
-      expect(
-        focusableCell.find('[data-test-subj="cell-content"]').text()
-      ).toEqual('0, A');
+      setTimeout(() => {
+        expect(
+          focusableCell.find('[data-test-subj="cell-content"]').text()
+        ).toEqual('0, A');
+      }, 0);
 
       findTestSubject(component, 'dataGridRowCell').at(3).simulate('focus');
-
-      // wait for a tick to give focus logic time to run
-      await act(async () => {
-        await new Promise((r) => setTimeout(r, 0));
-      });
-      component.update();
```

Comment:
This gets removed due to being in a `setTimeout`?

Reply:
Yeah, this doesn't work and was broken, even though it attempted to do the same thing (wait for the focus update)